site/app/libraries/database/DatabaseQueries.php

```diff
@@ -5158,14 +5158,51 @@ public function getNumberGradeInquiries($gradeable_id, $is_grade_inquiry_per_com
         return ($this->course_db->row()['cnt']);
     }
 
+    /**
+     * get the grader and the count of the gradeable component for inquiry
+     * If the grade is not-by component, function will return all graders who were involved in grading the student
+     * @param string $gradeable_id
+     * @param  bool $is_grade_inquiry_per_component_allowed
+     * @return array<string, int> of gcd_grader_id and count of unresolved grade inquiries of the grader
+     */
+    public function getGraderofGradeInquiry($gradeable_id, $is_grade_inquiry_per_component_allowed = true) {
+        $return = [];
+        if ($is_grade_inquiry_per_component_allowed) {
+            $this->course_db->query("SELECT count(b.*), b.gcd_grader_id FROM grade_inquiries a JOIN gradeable_component_data b ON a.gc_id = b.gc_id JOIN gradeable_data c ON b.gd_id = c.gd_id AND a.user_id = c.gd_user_id WHERE a.status = '-1' AND a.g_id=? GROUP BY b.gcd_grader_id", [$gradeable_id]);
```

Comment:
This line is very long and is difficult to read.  Also, the aliases don't really make sense.  Please change them to a character, or series of characters which makes sense given the context.

```suggestion
            $this->course_db->query("
                SELECT
                    count(b.*),
                    b.gcd_grader_id
                FROM grade_inquiries a
                INNER JOIN gradeable_component_data b ON (
                    a.gc_id = b.gc_id
                )
                INNER JOIN gradeable_data c ON (
                    b.gd_id = c.gd_id
                    AND a.user_id = c.gd_user_id
                )
                WHERE
                    a.status = -1
                    AND a.g_id = ?
                GROUP BY b.gcd_grader_id
            ", [$gradeable_id]);
```

Reply:
I agree that it is hard to read, and I did initially wrote with necessary space and indentation, yet the code style I referenced were mostly condensed to one line, which I thought was intentional. I will make adequate changes.